react/slider/src/style.css

```diff
@@ -0,0 +1,164 @@
+.wrapper {
+  display: grid;
+  width: 500px;
+  max-width: 100%;
+  margin: 8px 0px;
+}
+.input-wrapper {
+  grid-row: 1;
+  grid-column: 1;
+  display: grid;
+}
+input[type="range"] {
+  appearance: none;
+  height: 44px;
+  grid-row: 1;
+  grid-column: 1;
+  pointer-events: none;
+}
+input[type="range"]:focus {
+  outline: none;
+}
+input[type="range"]::-webkit-slider-thumb {
+  appearance: none;
+  width: 28px;
+  height: 28px;
+  background-color: var(--w-s-color-background-primary);
+  cursor: pointer;
+  transform: translateY(-12.5px);
+  border-radius: 14px;
+  pointer-events: all !important;
+}
+input[type="range"]:hover::-webkit-slider-thumb {
+  box-shadow: 0 0 0 6px #00000014;
+}
+input[type="range"]:focus::-webkit-slider-thumb {
+  box-shadow: 0 0 0 8px #00000014;
+}
+input[type="range"]:focus-visible::-webkit-slider-thumb {
+  outline: 2px solid #06befb;
```

Comment:
use token

Reply:
Id suggest we decleare all of these as local scope css vars ontop of the css.